src/Service.Tests/Multidab-config.CosmosDb_NoSql.json

```diff
@@ -0,0 +1,379 @@
+{
+  "$schema": "https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json",
+  "data-source": {
+    "database-type": "cosmosdb_nosql",
+    "connection-string": "AccountEndpoint=https://localhost:8081/;AccountKey=C2y6yDjf5/R\u002Bob0N8A7Cgv30VRDJIWEHLM\u002B4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==",
```

Comment:
maybe this was from auto generation, should the `\u002B` be there?

Reply:
Took this from existing dab-config.cosmos file.

Reply:
I see. https://github.com/Azure/data-api-builder/pull/1691 was supposed to address this, we may have to revisit in separate workstream. Closing this thread.